src/js_native_api_v8.cc

```diff
@@ -134,16 +134,13 @@ napi_status NewExternalString(napi_env env,
   return status;
 }
 
-class TrackedStringResource : public Finalizer, RefTracker {
+class TrackedStringResource : public TrackedFinalizer {
```

Comment:
I think `TrackedStringResource` didn't benefit from inheriting `TrackedFinalizer`. Its `Finalize` overrides `TrackedFinalizer::Finalize` completely.

Reply:
Thanks! The latest iteration has restored the `TrackedStringResource` to inherit directly from `RefTracker`. Also, the `Reference` and `TrackedFianlizer` are also inherited directly from `RefTracker`. It makes the inheritance chains a bit flatter.
(Sorry, it seems that this comment was not sent when I resolved the conversation originally.)